### DISCUSSION TITLE
Helm Chart Support and Release

### DISCUSSION THREAD

**Discussion by alexanderursu99** [apache/superset#12975] [General]
Hi, just wondering what the status is of the Helm chart for Superset. I'm evaluating Superset, and having an official Helm chart release would be very helpful for our production usage.

**Comment by amithapa** [comment]
When will the helm chart be released? I guess it would be super helpful to manage the deployment of superset.

**Comment by amitmiran137** [comment]
it is released now: https://github.com/apache/superset/releases/tag/superset-helm-chart-0.1.1
